docs/installation.md

```diff
@@ -291,12 +289,12 @@ unzip it, and then run:
 ./refresh-data.sh /path/to/dump.sql
 ```
 
-This will remove the initial Wagtail admin user that was created by
-the `initial-data.sh` script that was called by `backend.sh`.
-If you need to access the Wagtail admin, create a new user with the following:
+The `refresh-data.sh` script will apply the same changes as the `initial-data.sh` script described above (including setting up the `admin` superuser), but will not apply migrations.
```

Comment:
Thanks for the review! Yeah, maybe to simplify it does make sense to include migrations in `refresh-data.sh` as well. The use case I had in mind was one where you want to test the database as is; not how it would be with the most recent version of the code.